core/api/package.json

```diff
@@ -19,9 +19,9 @@
     "exporter": "yarn build && node lib/servers/exporter.js",
     "daily-notif": "yarn build && node lib/servers/daily-balance-notification.js",
     "test": "export JEST_JUNIT_OUTPUT_DIR=\"./artifacts\" && jest --ci --reporters=default --reporters=jest-junit",
-    "test:unit": ". ../../.env && LOGLEVEL=warn jest --config ./test/unit/jest.config.js --bail --verbose $TEST | yarn pino-pretty -c -l",
```

Comment:
Only one step left here so far (to re-add pino-pretty to the buck call)

Reply:
Don't see this as a strong requirement - output looks fine.